tools/docker/builder/openwrt/build.sh

```diff
@@ -97,7 +97,7 @@ main() {
         netgear-rax40)
             TARGET=intel_mips
```

Comment:
Commit message again... Please fix it to align with the CONTRIBUTING.md instructions as I showed in previous commits.

Reply:
changed the commit message.